packages/cli/src/LoadNodesAndCredentials.ts

```diff
@@ -183,8 +183,10 @@ export class LoadNodesAndCredentials implements INodesAndCredentials {
 			? `npm update ${packageName}`
 			: `npm install ${packageName}${options.version ? `@${options.version}` : ''}`;
 
+		const communityService = Container.get(CommunityService);
+
 		try {
-			await executeCommand(command);
+			await communityService.executeNpmCommand(command);
```

Comment:
Perhaps `installNpmModule`, `removeNpmModule`, and `updateNpmModule` should also move to `CommunityService`

Reply:
Turns out they're very much coupled together in those methods... Perhaps we need a dedicated `NpmService`? Since this PR is already big enough, we could address this separately.